internal/log/stdLogger.go

```diff
@@ -0,0 +1,51 @@
+package log
+
+import (
+	"fmt"
+	"io"
+	goLog "log"
+	"os"
+	"runtime"
+)
+
+var stdLogger StdLogger
+
+type StdLogger struct {
+	Logger
```

Comment:
Why are you embedding an interface? Looks like this should just be an empty struct{}:
```
type Standard struct{}
```

Reply:
While it only functionally only needs to be an empty struct, this makes it clear to someone viewing the file that the struct is intended to implement the interface.

Reply:
Sorry, somehow I missed this comment when I went through. This is not what you are doing here. What you are doing here is actually declaring a public Field called Logger. This field can be assigned to and called. If it's called before being given a non-nil value it will panic. None of the functions on the non-pointer version of the Struct will call any of the log functions, but the pointer version of the struct will attempt to call functions on the Logger field. 

This is not what we want. This should just be an empty struct.